### DISCUSSION TITLE
Update xDrip to the latest Nightly if you are using G7 or One+

### DISCUSSION THREAD

**Discussion by Navid200** [NightscoutFoundation/xDrip#3474] [Show and tell]
There has been an update to xDrip on May 22, 2024 that should improve connectivity with G7.
https://github.com/NightscoutFoundation/xDrip/releases/tag/2024.05.22

This explains how xDrip update works: https://navid200.github.io/xDrip/docs/Updates.html  

If you are using xDrip to collect from G7 or One+, you should use a release not older than May 22, 2024.

**Comment by bsw-del** [comment]
Thanks for the update @Navid200 , FWIW, I installed the update and was no longer getting readings to Xdrip, rebooted the phone after an hour of no readings, then my phone did the May Security Update, could not get Xdrip to get any readings at all. Tried uninstalling Xdrip and rolling back, didn't work. Maybe something in that May Security update effed things up?
Was getting Initialization errors on each connect. 
I moved over to G7 App for now. 
(Pixel 8 Pro)

**Comment by Navid200** [comment]
The May 22 release had an issue.  Considering all the developments since, is best to use an alternative.  
As of today, the best release for G7 is [June 1, 2024](https://github.com/NightscoutFoundation/xDrip/releases/tag/2024.06.01).
